tools/gulp/util/rollup-helper.ts

```diff
@@ -67,3 +84,27 @@ export function createRollupBundle(config: BundleConfig): Promise<any> {
 
   return rollup.rollup(bundleOptions).then((bundle: any) => bundle.write(writeOptions));
 }
+
+/**
+ * Function that will be used by rollup to detect imports that should be treated as external
+ * dependencies. This function also ignores the import shorthands from the NGC (tsickle)
+ **/
+function isExternalImport(moduleId: string): boolean {
+  return ROLLUP_EXTERNALS.indexOf(rewriteConvertedImports(moduleId)) !== -1;
+}
+
+/**
+ * When compiling with @angular/tsc-wrapped, shorthand imports will be converted into more
+ * explicit paths. This function rewrites all expanded imports to their original import.
+ * This is important, because when composing releases, the directories won't be present anymore.
```

Comment:
It's not an issue on `angular/angular`. It's a feature on tsickle. I was talking to @alexeagle about adding a flag or something so we can be more flexible about it.